package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "material-ui-json-schema-viewer",
+  "version": "1.0.0",
+  "main": "index.js",
+  "repository": "https://github.com/taskcluster/material-ui-json-schema-viewer",
+  "license": "MIT",
+  "scripts": {
+    "build": "neutrino build",
+    "start": "neutrino start",
+    "lint": "neutrino lint",
+    "test": "neutrino test",
+    "styleguide": "styleguidist server",
+    "styleguide:build": "styleguidist build"
```

Comment:
We could probably add the following:

```
"deploy": "styleguidist build && gh-pages --remote upstream -d styleguide",
```

This would run the styleguide build then push the changes to the gh-pages branch which would allow users to view the styleguide on https://taskcluster.github.io/material-ui-json-schema-viewer

^ isn't tested so you'll probably want to test that it works. Also, we'll need to add `gh-pages` to the list of devDependencies.

Reply:
Tested the styleguide deployment on the forked repo (with `origin`) and it seems to work 👍 
Included this script to package.json.

I also updated `gitignore` to ignore the `styleguide` build directory as well.

Reply:
> Should I also update gitignore to ignore the styleguide build directory?

Yep!

Reply:
Updated!